include/cling/Interpreter/Interpreter.h

```diff
@@ -142,6 +142,11 @@ namespace cling {
     ///
     InvocationOptions m_Opts;
 
+    ///\brief Back pointer to *this* which stores parent information.
```

Comment:
Indeed, after reading the whole patch, this is what I guess.  Could you still rephrase the doc string to be more explicit/clear?